SecurityReport.py

```diff
@@ -343,25 +344,27 @@ def populate_member_values(self, iot_objects_array):
 
             # Security Progress
             import variables
-            if iot_object.security_phase_progress == variables.NOT_STARTED_PHASE:
-                not_started_counter = not_started_counter + 1
-            if iot_object.security_phase_progress == variables.IN_PROGRESS_PHASE:
-                in_progress_counter = in_progress_counter + 1
             if iot_object.security_phase_progress == variables.DONE_PHASE:
                 done_counter = done_counter + 1
-            if iot_object.security_phase_progress == variables.NOT_EVALUATED_PHASE:
+            elif iot_object.security_phase_progress == variables.IN_PROGRESS_PHASE:
+                in_progress_counter = in_progress_counter + 1
+            elif iot_object.security_phase_progress == variables.ON_HOLD_PHASE:
+                on_hold_counter = on_hold_counter + 1

```

Comment:
Maybe we should use the counter_utils.py methods at some point on that and rely on the SecurityReport object instead of the iot_objects array.
I think that would make sense